mtf/metadata/README.md

```diff
@@ -0,0 +1,157 @@
+# Upstream Test Metedata PoC
```

Comment:
Metadata*

Reply:
thats true, it is hard to understand. it is very complex, But I would like to not invest to much effort to documentation, until, there will be wider agreement. it is on track. so this is more less PoC and something working for MTF